protobuf_definitions/message_formats.proto

```diff
@@ -443,6 +443,15 @@ enum Framerate {
   FRAMERATE_FPS_25 = 2; // 25 frames per second
 }
 
+/**
+  * Which camera to control.
+  */
+enum Camera {
+  CAMERA_UNSPECIFIED = 0; // Camera not specified
+  CAMERA_MAIN = 1; // main camera
```

Comment:
Nitpicking, but consistent capitalization as these will be used in documentation?

Reply:
Capitalization fixed in fa579ee786a0e6453ad7288004ca80e7677c3bb5